src/main/java/org/folio/rest/impl/SamlAPI.java

```diff
@@ -663,4 +660,12 @@ private boolean isInvalidOrigin(String origin) {
     return origin == null || origin.isBlank() || origin.trim().contentEquals("*");
   }
 
+  static String getCqlUserQuery(String userPropertyName, String value) {
+    String field;
+    switch (userPropertyName.toLowerCase()) {
+      case "email" : field = "personal.email"; break;
```

Comment:
For me the bug is in the front-end and should be fixed there:
https://github.com/folio-org/ui-tenant-settings/blob/master/src/constants.js

The other changes in mod-login-saml are fine.

In addition mod-login-saml should only allow field names that are on a whitelist (barcode, externalSystemId, id, username, personal.email) and fail with an error message for any other field name to prevent a full table scan.

Reply:
So we aren't fixing it here.. Alright. But still keep info about what mod-users supports.. Not DRY. OK